### ISSUE TITLE
feat(zebra-scan): Make the key storage database persistent

### ISSUE THREAD

**Issue by oxarbitrage** [ZcashFoundation/zebra#7928]
We have a RAM key database as part of https://github.com/ZcashFoundation/zebra/issues/7904

We want to make this persistent, this task is blocked on https://github.com/ZcashFoundation/zebra/issues/7904 and https://github.com/ZcashFoundation/zebra/issues/7926

**Comment by mpguerra** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Engineering-Team-5fb24d9264a3e8000e666a9e/issues/ZcashFoundation/zebra/7928?planning-poker) with Zenhub @arya2 @oxarbitrage @teor2345 @upbqdn

**Comment by teor2345** [comment]
This was already done in PR #8036
